client/post-editor/editor-drawer/index.jsx

```diff
@@ -152,39 +152,36 @@ const EditorDrawer = React.createClass( {
 				className="editor-drawer__accordion"
 			/>
 		);
-	},
+	}
 
-	renderSharing: function() {
+	renderSharing() {
 		return (
 			<AsyncLoad
 				require="post-editor/editor-sharing/accordion"
 				site={ this.props.site }
 				post={ this.props.post }
 			/>
 		);
-	},
+	}
 
-	renderFeaturedImage: function() {
+	renderFeaturedImage() {
 		if ( ! this.currentPostTypeSupports( 'thumbnail' ) ) {
 			return;
 		}
 
 		return (
 			<AsyncLoad require="./featured-image" site={ this.props.site } post={ this.props.post } />
 		);
-	},
+	}
 
-	renderExcerpt: function() {
-		let excerpt;
+	renderExcerpt() {
 		const { translate } = this.props;
 
 		if ( ! this.currentPostTypeSupports( 'excerpt' ) ) {
 			return;
 		}
 
-		if ( this.props.post ) {
-			excerpt = this.props.post.excerpt;
-		}
+		const excerpt = get( this.props.post, 'excerpt' );
```

Comment:
Nitpick: Would it be good to provide `''` as a default value for `get()`, since it'll otherwise return `undefined`, and `FormTextarea` excepts to be passed a string? That might not actually have any real consequences, and is probably very unlikely to occur, but I just thought I'd ask since the previous code explicitly checked for it.

Reply:
I think it's safe to set the value to `undefined`.
I had to re-jog my memory with regular `<textarea />`s, here's the warning we see when trying to set the value to `null` (one of my tries):

```jsx
<textarea value={ null } />
```
>>
```
Warning: `value` prop on `textarea` should not be null. 
Consider using the empty string to clear the component or `undefined` for uncontrolled components
```
I don't think that there are any issues with defaulting to an empty string, I tested it and the placeholder still shows.
I'd be happy to add the default but just as happy to leave it. I'll merge for now but we could add that later perhaps? :)

Reply:
Ah, yeah, it sounds like it's totally fine to leave it the way it is then. Thanks for checking :)